runtime/layers/function/bootstrap

```diff
@@ -1,11 +1,35 @@
-#!/bin/sh
+#!/opt/bin/php
+<?php declare(strict_types=1);
 
-# Fail on error
-set -e
+use Bref\Runtime\LambdaRuntime;
 
-while true
-do
-    # We redirect stderr to stdout so that everything
-    # written on the output ends up in Cloudwatch automatically
-    /opt/bin/php $LAMBDA_TASK_ROOT/$_HANDLER 2>&1
-done
+ini_set('display_errors', '1');
+error_reporting(E_ALL);
+
+$appRoot = getenv('LAMBDA_TASK_ROOT');
+
+/** @noinspection PhpIncludeInspection */
+require $appRoot . '/vendor/autoload.php';
+
+$lambdaRuntime = LambdaRuntime::fromEnvironmentVariable();
+
+$handlerFile = $appRoot . '/' . getenv('_HANDLER');
+if (! is_file($handlerFile)) {
+    $lambdaRuntime->failInitialization("Handler `$handlerFile` doesn't exist");
+}
+
+/** @noinspection PhpIncludeInspection */
+$handler = require $handlerFile;
+
+if (! $handler) {
+    $lambdaRuntime->failInitialization("Handler `$handlerFile` must return a function or Handler object");
+}
+
+$loopMax = getenv('BREF_LOOP_MAX') ?: 1;
+$loops = 0;
+while (true) {
+    if (++$loops > $loopMax) {
+        exit(0);
+    }
+    $lambdaRuntime->processNextEvent($handler);
+}
```

Comment:
This is the core of the new runtime: by default we do only 1 loop iteration and we terminate. That allows preserving the classic PHP behavior of restarting a new process every time.

Simply setting the `BREF_LOOP_MAX` to something else will cause the runtime to keep the PHP process alive, increasing performances.